### PR TITLE
Stash rockon initial checkin

### PR DESCRIPTION
**Pull request by tezatop**
Stash is a self-hosted webapp written in Go which organizes and serves your porn.
homepage: https://stashapp.cc

> To ease and accelerate the PR submission, please use the template below to provide all requested information.
> You can find guidelines on which docker image to use in the [Rockstor's documentation](http://rockstor.com/docs/contribute_rockons.html#which-docker-image-should-i-use), 
> as well as examples of proper formatting in other rock-ons ([here](https://github.com/rockstor/rockon-registry/blob/master/teamspeak3.json)
> and [here](https://github.com/rockstor/rockon-registry/blob/master/nextcloud-official.json))

Fixes # .

### General information on project
This pull request proposes to add a new rock-on for the following project:
- name: Stash
- website: https://stashapp.cc/
- description: Stash is a self-hosted webapp written in Go which organizes and serves your porn.

### Information on docker image
- docker image: https://hub.docker.com/r/stashapp/stash
- is an official docker image available for this project?: yes, this is the one used

### Checklist
- [x] Passes [JSONlint](https://jsonlint.com) validation
- [x] Entry added to `root.json` in _alphabetical_ order (for new rock-on only)
- [x] `"description"` object lists and links to the docker image used
- [x] `"description"` object provides information on the image's particularities (advantage over another existing rock-on for the same project, for instance)
- [x] `"website"` object links to project's main website

### CODE REVIEW

**Comment by Hooverdan96** [comment]
tried a quick test, however it failed during the install, there seems to be an issue with starting up the container (no container named 'stash' available). Not sure, what the root cause is on this.

**Comment by tezatop2** [comment]
I installed a fresh rockstor install in vbox and followed [These instructions](https://forum.rockstor.com/t/how-to-test-a-rock-on-pull-request/5224), and didn't take any shortcuts (i only did wget the stash.json from this pull request to rockon-metastore, not the root.json), -and the install went well, so i can't figure out what went wrong for @Hooverdan96, maybe someone could give this pull request another go?

PS.
(i lost access to my old e-mail account, so the 2-step verification for logging in to github prevents me from using my old github account)

File: stash.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          },
+          "PGID": {
+            "description": "Enter a valid GID to use along with the above UID. It (or the above UID) must have full permissions to all Shares mapped in the previous step.",
+            "label": "GID",
+            "index": 3
+          }
+        }
+      }
+    },
+    "description": "Stash is a self-hosted webapp written in Go which organizes and serves your porn.<br><br> Stash gathers information about videos in your collection from the internet, and is extensible through the use of community-built plugins for a large number of content producers and sites.<br> Stash supports a wide variety of both video and image formats.<br> You can tag videos and find them later.<br> Stash provides statistics about performers, tags, studios and more.",
```
Comment (Hooverdan96) [inline stash.json:62]:
```In order to align with our attempt to harmonize the tags displayed, I suggest you add this to the Description:
    "description": "<p>Stash is a self-hosted webapp written in Go which organizes and serves your porn.</p><br><p> Stash gathers information about videos in your collection from the internet, and is extensible through the use of community-built plugins for a large number of content producers and sites.<br>Stash supports a wide variety of both video and image formats. You can tag videos and find them later.<br>Stash provides statistics about performers, tags, studios and more.</p><p>Based on the official docker image: <a href='https://hub.docker.com/r/stashapp/stash' target='_blank'>https://hub.docker.com/r/stashapp/stash<a>, available for amd64 and arm64 architecture.</p>",
```

Comment (Hooverdan96) [reply stash.json:62]: In order to keep with the architecture tagging, I am suggesting to add the line about the source of the image, and supported architecture. I added some paragraph tags and combined two features into a single line to reduce the footprint a little bit.

**Review by Hooverdan96 — CHANGES_REQUESTED** [review]

**Comment by Hooverdan96** [comment]
I finally got it to work, as I realized something about "reusing" shares in the same rockon install. I think the only suggestion I have is the adding of the image source and supported architecture line that we now have for all of the other Rockons, and also a slight reduction in the number of lines for the description like this (not sure my "review" is capturing that accurately:
`
"description":"<p>Stash is a self-hosted webapp written in Go which organizes and serves your porn.</p><br><p> Stash gathers information about videos in your collection from the internet, and is extensible through the use of community-built plugins for a large number of content producers and sites.<br>Stash supports a wide variety of both video and image formats. You can tag videos and find them later.<br>Stash provides statistics about performers, tags, studios and more.</p><p>Based on the official docker image: <a href='https://hub.docker.com/r/stashapp/stash' target='_blank'>https://hub.docker.com/r/stashapp/stash<a>, available for amd64 and arm64 architecture.</p>"
`

File: stash.json (hunk 1 above)
Comment (tezatop2) [reply stash.json:62]: yeah, i think the rules about the source of the image was not in the description i readd whn creating the rockson, but i think maybe the requirement was added around then.

Comment (Hooverdan96) [reply stash.json:62]: That could very well be.
Will you be able to update the description similar to what I posted below that includes the architecture reference as well?

Thanks

**Comment by phillxnet** [comment]
Closing as requested by revised GitHub account @tezatop.

**Pull request closed without merging**